ci/e2e.sh

```diff
@@ -36,8 +36,8 @@ echo "Add keys for chains"
 echo "-----------------------------------------------------------------------------------------------------------------"
 hermes -c "$CONFIG_PATH" keys add "$CHAIN_A" -f user_seed_"$CHAIN_A".json
 hermes -c "$CONFIG_PATH" keys add "$CHAIN_B" -f user_seed_"$CHAIN_B".json
-hermes -c "$CONFIG_PATH" keys add "$CHAIN_A" -f user2_seed_"$CHAIN_A".json  -k user2
-hermes -c "$CONFIG_PATH" keys add "$CHAIN_B" -f user2_seed_"$CHAIN_B".json  -k user2
+hermes -c "$CONFIG_PATH" keys add "$CHAIN_A" -f user2_seed_"$CHAIN_A".json  -n user2
```

Comment:
Should it be `--key-name` ?

Reply:
After running
- `cargo build --release --bin hermes`
- `./target/release/hermes keys add -h`
It's indeed renamed (again?) to `--key-name`

Code updated

Reply:
Well if it works already on master then it's the tutorial's fault
I will close this PR now